sirepo/pkcli/job_agent.py

```diff
@@ -644,19 +645,39 @@ async def _prepare_simulation(self):
                 opName=job.OP_ANALYSIS,
             ),
             send_reply=False,
+            op_id=self.msg.opId,
```

Comment:
why was this added?

Reply:
[pkdebug_str() expects its](https://github.com/radiasoft/sirepo/blob/5295ae9b2762c2537a5d24e8b613f2a9c6e4982c/sirepo/pkcli/job_agent.py#L466)